example/src/main/java/com/spruceid/mobilesdkexample/wallet/AchievementCredentialItem.kt

```diff
@@ -244,12 +245,15 @@ class AchievementCredentialItem {
 
     @Composable
     fun detailsComponent() {
-        val awardedDate = keyPathFinder(credential, mutableListOf("awardedDate")).toString()
+        println("credential: $credential")
+        // NOTE: The credential contains a `vc` property with the verifiable credential payload.
+        val awardedDate = keyPathFinder(credential, mutableListOf("vc", "awardedDate")).toString()
+        println("awardedDate: $awardedDate")
```

Comment:
NOTE: The example SD-JWT (see `Utils.kt`) that I am using includes a `vc` claims field, of which `awardedDate` is a sub property.

This may be incompatible with other example credentials, so need to check that we may want to first extract the `vc` payload if it exists, then process the payload.